parser/sql/dialect/oracle/src/main/antlr4/imports/oracle/DALStatement.g4

```diff
@@ -30,3 +30,32 @@ dbLinkInfo
 explain
     : EXPLAIN PLAN (SET STATEMENT_ID EQ_ stringLiterals)? (INTO (schemaName DOT_)? tableName (AT_ dbLinkInfo)? )? FOR (insert | delete | update | select)
     ;
+
+parameterName
+    : identifier
+    ;
+
+showErrors
```

Comment:
According to oracle doc - https://docs.oracle.com/en/database/oracle/oracle-database/19/sqpug/SHOW.html#GUID-6BB1499D-E537-43D1-A209-401F5DB95E16, It seems that these are all part of the show statement, except that it has different options.

Reply:
ok, I will complete all the options for show commend